src/PrestaShopBundle/Resources/config/routing/admin/improve/international/country.yml

```diff
@@ -0,0 +1,42 @@
+#todo uncomment all legacy links when page is finished.
+admin_countries_index:
+  path: /
+  methods: GET
+  defaults:
+    _controller: 'PrestaShopBundle\Controller\Admin\Improve\International\CountryController::indexAction'
+    _legacy_controller: AdminCountries
+#    _legacy_link: AdminCountries
+
+admin_countries_search:
+  path: /
+  methods: POST
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\CommonController::searchGridAction
+    gridDefinitionFactoryServiceId: prestashop.core.grid.definition.factory.country
+    redirectRoute: admin_countries_index
+    _legacy_controller: AdminCountries
+#    _legacy_link: AdminCountries:submitFiltercountry
+
+admin_country_edit:
+  path: /{countryId}/edit
+  methods: [ GET, POST ]
+  defaults:
+    _controller: 'PrestaShopBundle\Controller\Admin\Improve\International\CountryController::editAction'
+    _legacy_controller: AdminCountry
+#    _legacy_link: AdminCountry:updateCountry
```

Comment:
You can run the linter locally with `php bin/console lint:yaml src` to make sure everything is ok before pushing it It's faster than waiting for the CI from github

Reply:
I don't really get the problem here. I can't see any warnings, its how every other yml is commenting and if i remove the space after uncommenting the line it will have bad spacing so not sure why we need to remove the space here or where i can see the warnings. The command lint:yaml doesn't say anything about this line.

Reply:
Actually I looked at the wrong command it's actually `yamllint -c .github/workflows/yamllint/.yamllint.yml src` which had this warning but it seems all good now